net-libs/libnice/libnice-0.1.13.recipe

```diff
@@ -0,0 +1,73 @@
+SUMMARY="A library for establishing peer-to-peer UDP data streams"
+DESCRIPTION="Libnice is an implementation of the IETF's Interactive .
+Connectivity Establishment (ICE) standard (RFC 5245) and the Session Traversal .
+Utilities for NAT (STUN) standard (RFC 5389) .
+
+ICE is useful for applications that want to establish peer-to-peer UDP .
+data streams. It automates the process of traversing NATs and provides .
+security against some attackes. It also allows applications to create .
+reliable streams using a TCP over UDP layer."
+HOMEPAGE="http://nice.freedesktop.org/wiki"
+COPYRIGHT="2006-2011 Collabora Ltd.
+    2006-2011 Nokia Corporation"
+LICENSE="MPL v1.1
+    GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="http://nice.freedesktop.org/releases/libnice-0.1.13.tar.gz"
+CHECKSUM_SHA256="61112d9f3be933a827c8365f20551563953af6718057928f51f487bfe88419e1"
+PATCHES="libnice_x86-0.1.13.patchset"
+
+ARCHITECTURES="x86_gcc2 x86"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+    libnice$secondaryArchSuffix = $portVersion
+    devel:libnice$secondaryArchSuffix = $portVersion
+    lib:libnice$secondaryArchSuffix = $portVersion
+    lib:libstun$secondaryArchSuffix = $portVersion
+    cmd:sdp_example$secondaryArchSuffix = $portVersion
+    cmd:simple_example$secondaryArchSuffix = $portVersion
+    cmd:stunbdc$secondaryArchSuffix = $portVersion
+    cmd:stund$secondaryArchSuffix = $portVersion
+    cmd:threaded_example$secondaryArchSuffix = $portVersion
+    "
+REQUIRES="
+    haiku$secondaryArchSuffix
+    lib:libgio_2.0$secondaryArchSuffix
+    lib:libgmodule_2.0$secondaryArchSuffix
+    lib:libz$secondaryArchSuffix
+    lib:libgobject_2.0$secondaryArchSuffix
+    lib:libffi$secondaryArchSuffix
+    lib:libgthread_2.0$secondaryArchSuffix
+    lib:libglib_2.0$secondaryArchSuffix
+    lib:libintl$secondaryArchSuffix
+    lib:libiconv$secondaryArchSuffix
+    "
+    
+BUILD_REQUIRES="
+    haiku${secondaryArchSuffix}_devel
+    devel:libglib_2.0$secondaryArchSuffix
```

Comment:
from the libnice documentation, only glib is required for building, the rest of the lib are those that haikuporter complained about when building, those are runtime requirements if I'm not wrong